models/public/hrnet-v2-c1-segmentation/model.yml

```diff
@@ -0,0 +1,136 @@
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/models/mobilenet.py
+  - name: mit_semseg/models/models.py
+    size: 21185
+    sha256: fea2849abca35a260a64e852397bcccad64e2fd495fad668ad1622b5080033eb
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/models/models.py
+  - name: mit_semseg/models/resnet.py
+    size: 6770
+    sha256: 685bb02446ce28eb1b110f3dfa9f6534a2ea34f50f09be0d7c165bd2c5c6c899
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/models/resnet.py
+  - name: mit_semseg/models/resnext.py
+    size: 5367
+    sha256: 271a95bdabb9e2c00ecc2e6765b7521f708cda5312b4d78bfb895374f3509044
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/models/resnext.py
+  - name: mit_semseg/models/utils.py
+    size: 577
+    sha256: 55a8f41269be28c17e456e5ae1f246bf71354765739dceca1e45becd58cba5e2
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/models/utils.py
+  - name: mit_semseg/__init__.py
+    size: 63
+    sha256: 7e58a465dbf83903f13d94ffce5b3353bcc8eb2154cd568bb405341f8a02ce5e
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/__init__.py
+  - name: ckpt/decoder_epoch_30.pth
+    size: 4780389
+    sha256: 6060764940b620804befaca4df916de019c82c4e96ace00916991e223af37d1e
+    source: http://sceneparsing.csail.mit.edu/model/pytorch/ade20k-hrnetv2-c1/decoder_epoch_30.pth
+  - name: ckpt/encoder_epoch_30.pth
+    size: 262518297
+    sha256: 0bba1a5da1484b21a42f89e05459e434119c829d6aae64d1c6d3ae0c38d2b43c
+    source: http://sceneparsing.csail.mit.edu/model/pytorch/ade20k-hrnetv2-c1/encoder_epoch_30.pth
+postprocessing:
+  - $type: regex_replace
+    file: mit_semseg/models/models.py
+    pattern: 'segSize=None'
+    replacement: 'segSize=320'
+  - $type: regex_replace
+    file: mit_semseg/models/models.py
+    pattern: 'feed_dict\[\Wimg_data\W\]'
+    replacement: 'feed_dict'
+  - $type: regex_replace
+    file: mit_semseg/models/models.py
+    pattern: '= net_enc'
+    replacement: '= eval(net_enc)'
+  - $type: regex_replace
+    file: mit_semseg/models/models.py
+    pattern: '= net_dec'
+    replacement: '= eval(net_dec)'
+conversion_to_onnx_args:
+  - --model-path=$dl_dir
+  - --model-name=SegmentationModule
+  - --import-module=mit_semseg.models
+  - --input-shape=1,3,320,320
+  - --output-file=$conv_dir/hrnet-v2-c1-segmentation.onnx
+  - --model-param=net_enc=r'ModelBuilder.build_encoder("hrnetv2", 720, r"$dl_dir/ckpt/encoder_epoch_30.pth")'
```

Comment:
you use raw string here due to eval issue on Windows. Is my understanding correct?
Do you test this aproach on windows machine? Does it really work?

Reply:
Yes, it does. I test it on Windows, prefix 'r' doesn't let eval to use escape symbols

Reply:
great!